ext/settings.html

```diff
@@ -3213,6 +3221,87 @@ <h1>Yomitan Settings</h1>
     </div>
 </div></div>
 
+<!-- Generate anki deck modal -->
+<div id="generate-anki-deck-modal" class="modal" tabindex="-1" role="dialog" hidden><div class="modal-content modal-content-full">
+    <div class="modal-header"><div class="modal-title">Anki Deck Generator</div></div>
+    <div class="modal-body generate-anki-deck-layout">
+        <div class="generate-anki-deck-info">
+            <p class="warning-text">
+                WARNING: This feature is experimental!
+            </p>
+            <p>
+                Enter a newline separated list of terms below to send directly to an Anki deck or export to an Anki deck file in <code>Notes in plain text (.txt)</code> format.
+            </p>
+        </div>
+        <textarea autocomplete="off" spellcheck="false" id="generate-anki-deck-textarea" class="no-wrap margin-above" data-tab-action="indent,4"></textarea>
+        <div class="generate-anki-deck-test-container margin-above">
+            <p>
+                Active Anki deck: <code id="generate-anki-deck-active-deck"></code><br>
+                Active Anki card format: <code id="generate-anki-deck-active-card-format"></code>
+            </p>
+            <div class="generate-anki-deck-test-table margin-above">
+                <div class="generate-anki-deck-test-table-header">Test word</div>
+                <div></div>
+                <input type="text" id="generate-anki-deck-test-text-input" class="form-control" value="読め" placeholder="Preview text" autocomplete="off" lang="ja">
```

Comment:
How hard is it to not hardcode ja here?

Reply:
Added this, it will set the example text based on language when the page is loaded or reloaded. Also worth noting the Anki Card Templates modal also has this issue (someone should go copy paste 6c635b9 into there as well)